frontend/ios/DroidKaigi 2019/Scenes/Announcement/AnnouncementTableViewCell.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  AnnouncementTableViewCell.swift
+//  DroidKaigi 2019
+//
+//  Created by 佐々木美穂 on 2019/02/02.
+//
+
+import UIKit
+
+class AnnouncementTableViewCell: UITableViewCell, Reusable {
```

Comment:
You don't need to implement `Reusable` if you don't use `UITableView+Extension`.